### PR TITLE
support multi-scheduler

### PR DESCRIPTION
**Pull request by huone1**
Signed-off-by: huone1 <huwanxing@huawei.com>
issue #1322

### CODE REVIEW

File: pkg/webhooks/admission/pods/mutate/annotation.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+	annotations := getAnnotation(resGroupConfig)
+	klog.V(3).Infof("annotations : %v", annotations)
+	for key, annotation := range annotations {
+		if pod.Annotations[key] == annotation {
+			return true
+		}
+	}
+
+	if resGroupConfig.Object.Key == "" && pod.Annotations["volcano.sh/resource-group"] == resGroupConfig.ResourceGroup {
```
Comment (wpeng102) [inline pkg/webhooks/admission/pods/mutate/annotation.go:69]: make "volcano.sh/resource-group" as a cost string

Comment (huone1) [reply pkg/webhooks/admission/pods/mutate/annotation.go]: Added a const string : defaultAnnotationKey = "volcano.sh/resource-group"

**Review by wpeng102 — APPROVED** [review]
/lgtm
/approve

**Review by shinytang6 — APPROVED** [review]
/approve

**Review by shinytang6 — APPROVED** [review]

**Pull request merged**